VideoLocker/src/main/java/org/edx/mobile/util/MediaConsentUtils.java

```diff
@@ -14,6 +16,8 @@
  */
 public class MediaConsentUtils {
 
+    @Inject Config config;
```

Comment:
This could be done everywhere :wink: but only for the platform name. It's fine as it is now I guess.